pages/accountLists/[accountListId]/coaches.page.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from 'react';
```

Comment:
Can you rename this file to `coaching.page.tsx` too? Just to match everything else. The main nav might need a corresponding update.